11ty/definitions/mansplaining.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Mansplaining
```

Comment:
```suggestion
title: Mansplain
```

Reply:
Changing this to make your part of speech correct! 🤗 Alternately, you can keep mansplaining and change it to a noun, your choice!

Eventually we'll get more sophisticated with the dictionary to include the other, related forms of speech for each word!